cmstestsuite/Tests.py

```diff
@@ -239,66 +254,65 @@
 
     Test('oom-static',
          task=batch_stdio, filenames=['oom-static.%l'],
-         languages=NON_INTERPRETED_LANGUAGES,
+         languages=(LANG_C, LANG_CPP, LANG_CPP14,
```

Comment:
If we add LANG_CPP23 in the future, how do we ensure we'll remember to put it here too?

Reply:
Honestly it doesn't really matter, having this test in a single language would probably be more than enough...

Reply:
That can be said for many of these tests (after all we're really testing the behavior of the sandbox). Still it makes sense to keep them uniform: maybe we add a new language where it's quite natural to implement `oom-static.%l` and I think it makes sense to do it.

Personally I would go with the same exclusion approach you did for MANAGER and COMMUNICATION.

Reply:
I disagree, and I would rather modify this test to just test C than follow that approach.

It's also very much not true that we're just testing the behaviour of the sandbox in these tests.

Reply:
Said otherwise: *some* tests (such as this one) test the behaviour of the sandbox.
Other tests don't.

Reply:
Yep that's what I said :)

Using only LANG_C isn't a bad idea, also for some other tests, but I guess for now we can keep it as it is.

I would still prefer to at least have this list defined above instead of inlined here if we keep it as it is, but it's not a strong opinion.